spec/controllers/api/v1/project_preferences_controller_spec.rb

```diff
@@ -137,4 +137,34 @@
 
     it_behaves_like "is creatable"
   end
+
+  describe "#update_settings", :focus do
+    let!(:project) { create(:project, owner: authorized_user) }
+    let!(:upp) { create(:user_project_preference, project: project) }
+    let(:settings_params) do
+      {
+        user_id: upp.user_id,
+        project_id: project.id,
+        settings: { workflow_id: 1234 }
+      }
+    end
+
+    let(:scopes) { %w(project_preference) }
```

Comment:
this is the problem why you can't get into the before_action method block, basically this has to marry the scopes in the require_authentication dsl block define in the controller [here](https://github.com/zooniverse/Panoptes/blob/18a44e0c7e6775c4a9ae1b794ce87ed86e485492/app/controllers/api/v1/project_preferences_controller.rb#L4)